src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.stories.tsx

```diff
@@ -22,7 +22,7 @@ const basePublisher = {
 };
 
 const baseArgs: Omit<IProposalDataListItemStructureProps, 'result' | 'publisher'> = {
-    date: '5 days left',
+    date: 1719963030308,
```

Comment:
Yes, that was because of the `draft` status, updated to `active`:
<img width="1008" alt="image" src="https://github.com/aragon/ods/assets/172602066/00264e76-f5f2-47dd-9edc-64cc8ac32179">